packages/typegpu/src/data/attributes.ts

```diff
@@ -169,7 +176,7 @@ export function size<TSize extends number, TData extends AnyData>(
  * Assigns an explicit numeric location to a struct member or a parameter that has this type.
  *
  * @example
- * const Data = d.ioStruct({
+ * const Data = d.struct({
```

Comment:
is this actually correct? or should it be `VertexOutput = {...`?

Reply:
It's technically correct, as regular struct fields can also have location attributes, but yeah, a more up-to-date doc would be using `VertexOutput`. Would you mind updating it? 🙌

Reply:
done